### DISCUSSION TITLE
Rounded Corners in JAVA

### DISCUSSION THREAD

**Discussion by ricardo-mrgd** [ImaginativeShohag/Why-Not-Image-Carousel#66] [Q&A]
Hi, I can't customize layout with cardview corners rounded in JAVA. Help please!

**Comment by ImaginativeShohag** [comment]
Please check out the [samples](https://github.com/ImaginativeShohag/Why-Not-Image-Carousel/tree/master/sample). [Here](https://github.com/ImaginativeShohag/Why-Not-Image-Carousel/blob/7566ed2cc78549089079fc3c32e3b2f0f2613b77/sample/src/main/res/layout/item_custom_fixed_size_layout_1.xml) is a layout with a rounded card view.

The solution is not connected with Java or Kotlin. You can change the corner radius from the XML layout file.
